### ISSUE TITLE
why does the app ask for password for user ( not system wide ) installed flatpak apps?

### ISSUE THREAD

**Issue by rezad1393** [vinifmor/bauh#163]
why does the app ask for password for user ( not system wide ) installed flatpak apps?
version : 0.9.11 
os: archlinux

**Comment by vinifmor** [comment]
Hi @rezad1393 , could you explain your scenario better ? (did it ask for you password when trying to install a new Flatpak at the user level ?) 

I've just tested here, and installed a Flatpak at the user level without any password prompt. Bauh only asks for you password if a given task requires it (e.g: sync package databases when Arch support is enabled)

**Comment by rezad1393** [comment]
sorry I forgot to put update at the end.
when I run this app it refreshes apps and shows me the list of upgradable flatpak apps.
when I press the upgrade button it asks for root password.
but using command line update from flatpak update doesnt need root password. (I dont have any system-wide installed flatpaks)

**Comment by vinifmor** [comment]
@rezad1393 , I've just tested upgrading a Flatpak at the user level, and bauh did not ask for my password. Perhaps you were about to upgrade other types of packages together that required password ? (like Arch or Snap packages)
e.g:
- gedit (flatpak/user): 1.0 -> 2.0
- firefox (arch): 2.0 -> 3.0
- vlc (snap): 1.1 -> 1.2

A simple test that can be done on your end is to downgrade a Flatpak at the user level, and then request to upgrade it.

**Comment by rezad1393** [comment]
I used your app to try to test with downgrade flatseal app then used upgrade and sure enough it asks for password.
I checked the setting and I have only selected flatpak to be managed by bauh and archlinux and appimages are not selected.

I removed the bauh config folder and run bauh and at the start it asked for root password.
then I removed the arch and appimage from options and now it seem to have fixed it.
